src/Symfony/Bridge/Twig/Tests/DataCollector/TwigDataCollectorTest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Symfony\Bridge\Twig\Tests\DataCollector;
+
+use Symfony\Bridge\Twig\DataCollector\TwigDataCollector;
+use Twig_Profiler_Profile;
+
+class TwigDataCollectorTest extends \PHPUnit_Framework_TestCase
```

Comment:
it does not test anything. Such things should be checked using something like PHP codesniffer, PHP CS fixer or other linters, tests are testing behaviour/public API.